docs/components/scroll-restoration.md

```diff
@@ -81,6 +104,32 @@ Or you may want to only use the pathname for some paths, and use the normal beha
 />
 ```
 
+## `scrollContainer`
+
+By default, this component will capture/restore scroll positions on `window`. If your UI scrolls via a different container element with `overflow:scroll`, you will want to use that instead of `window`. You can specify that via the `scrollContainer` prop using a selector or a `ref`:
```

Comment:
I don't know if there's simple guidance about whether to prefer ref or selector that fits in a sentence, but if there is, it would be good to add it. I feel `ref` is safer and easier to be confident about. But maybe people can figure that out.